### ISSUE TITLE
uploading and downloading a hashcode container corrupts the manifest.xml

### ISSUE THREAD

**Issue by mbakhoff** [open-eid/SiGa#21]
I'm signing an asice container that contains a xlsx file. SiGa successfully signs the file but digidoc desktop app reports the signature as invalid. 

The validation error looks something like this:
> Manifest datafile 'test.xlsx' mime 'application/octet-stream' does not match signature mime 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'.

My initial asice manifest.xml contains
```
<manifest:file-entry manifest:full-path="test.xlsx" manifest:media-type="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" />
```

The container is uploaded using /upload/hashcodecontainers and the signed version is downloaded using /hashcodecontainers/{containerId}. 

The manifest.xml returned in SiGa hashcode container contains:
```
<manifest:file-entry manifest:full-path="test.xlsx" manifest:media-type="application/octet-stream"/>
```

The manifest's media-type has clearly changed. The signature generated by SiGa correctly contains the mime-type from the initial manifest: `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet`. 

The issue was only noticed recently. I suspect it is related to dss/digidoc4j version upgrades. Please see commit 
https://github.com/open-eid/SiGa/commit/41c22e9cac0cd2e0eb047445df7fb271a6fb6ac1 and https://github.com/open-eid/digidoc4j/commit/f64f0f590dcff6a35f8848fb1b985073c1220d11

Perhaps ee.openeid.siga.service.signature.hashcode.HashcodeContainer#convertDataFiles should be changed to use MimeTypeUtil.fromMimeTypeString?

**Comment by naare** [comment]
Thank you for reporting it, we are investigating.

**Comment by jukutr** [comment]
This should be fixed now in SiGa v2.10.2.

**Comment by mbakhoff** [comment]
Seems to be fixed in demo env. Thanks for the quick response and fix!
